test/WebSites/ControllersFromServicesClassLibrary/ControllersFromServicesClassLibrary.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
```

Comment:
Looks like this missing the `preserveCompilationContext` flag from the project.json. I think project.json required an executable to produce a deps file.